portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -1418,5 +1454,19 @@ public Site getSite(String siteId) throws IdUnusedException
 			throw e;
 		}
 	}
+	
+	public String getRole(String siteId, String userId) {
+		String role = "";
+		try {
+			if(null!=getAuthzGroupService()) {
```

Comment:
In some scenarios getAuthzGroupService() comes null, but I have tested that with my previous validation:
!userId.equals(ADMIN_SITE)
It can be deleted.

Thank you.